src/verify/PhoneVerificationStartScreen.tsx

```diff
@@ -0,0 +1,293 @@
+import { defaultCountryCodeSelector, e164NumberSelector } from 'src/account/selectors'
+import { showError } from 'src/alert/actions'
+import { ErrorMessages } from 'src/app/ErrorMessages'
+import { registrationStepsSelector } from 'src/app/selectors'
+import BackButton from 'src/components/BackButton'
+import Button, { BtnTypes } from 'src/components/Button'
+import Dialog from 'src/components/Dialog'
+import KeyboardAwareScrollView from 'src/components/KeyboardAwareScrollView'
+import PhoneNumberInput from 'src/components/PhoneNumberInput'
+import TextButton from 'src/components/TextButton'
+import i18n from 'src/i18n'
+import { setHasSeenVerificationNux } from 'src/identity/actions'
+import { HeaderTitleWithSubtitle } from 'src/navigator/Headers'
+import { navigate, navigateHome } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { TopBarTextButton } from 'src/navigator/TopBarButton'
+import { StackParamList } from 'src/navigator/types'
+import { waitUntilSagasFinishLoading } from 'src/redux/sagas'
+import colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { getCountryFeatures } from 'src/utils/countryFeatures'
+import { getPhoneNumberState } from 'src/verify/utils'
+import { walletAddressSelector } from 'src/web3/selectors'
+
+function PhoneVerificationStartScreen({
+  route,
+  navigation,
+}: StackScreenProps<StackParamList, Screens.VerificationEducationScreen>) {
+  const [showLearnMoreDialog, setShowLearnMoreDialog] = useState(false)
+  const [showSkipDialog, setShowSkipDialog] = useState(false)
+  const [phoneNumberInfo, setPhoneNumberInfo] = useState(() =>
+    getPhoneNumberState(
+      cachedNumber || '',
+      cachedCountryCallingCode || '',
+      route.params?.selectedCountryCodeAlpha2 || RNLocalize.getCountry()
+    )
+  )
+
+  const { t } = useTranslation()
+  const dispatch = useDispatch()
+  const headerHeight = useHeaderHeight()
+
+  const account = useSelector(walletAddressSelector)
+  const cachedNumber = useSelector(e164NumberSelector)
+  const cachedCountryCallingCode = useSelector(defaultCountryCodeSelector)
+  const walletAddress = useSelector(walletAddressSelector)
+  const { step, totalSteps } = useSelector(registrationStepsSelector)
+
+  const countries = useMemo(() => new Countries(i18n.language), [i18n.language])
+  const country = phoneNumberInfo.countryCodeAlpha2
+    ? countries.getCountryByCodeAlpha2(phoneNumberInfo.countryCodeAlpha2)
+    : undefined
+
+  const onPressStart = async () => {
+    if (!canUsePhoneNumber()) {
+      return
+    }
+    // TODO figure out what this nux thing does
+    dispatch(setHasSeenVerificationNux(true))
+    // TODO dispatch action to call start verification service
+    navigate(Screens.VerificationInputScreen, {
+      registrationStep: route.params?.hideOnboardingStep ? undefined : { step, totalSteps },
```

Comment:
this fixes a pet peeve bug of mine, where if you verify the phone number after onboarding the registration steps show something like "step 4 of 3" on the input screen...that part shouldn't be there at all